### PR TITLE
Raises PipelineException on FillMaskPipeline when there are != 1 mask_token in the input

### PR DESCRIPTION
**Pull request by mfuntowicz**

### CODE REVIEW

File: src/transformers/pipelines.py [hunk 1]
```diff
@@ -808,6 +820,20 @@ def __init__(
 
         self.topk = topk
 
+    def ensure_exactly_one_mask_token(self, masked_index):
```
Comment (julien-c) [inline src/transformers/pipelines.py:823]: nit: Type for `masked_index `

File: src/transformers/pipelines.py [hunk 2]
```diff
@@ -808,6 +820,20 @@ def __init__(
...
+        if masked_index.numel() > 1:
+            raise PipelineException(
+                "fill-mask",
+                self.model.base_model_prefix,
+                "More than one mask_token ({}) is not supported".format(self.tokenizer.mask_token),
```
Comment (julien-c) [inline src/transformers/pipelines.py:828]: nit: use f-strings?

**Review by julien-c — APPROVED** [review]

File: src/transformers/pipelines.py [hunk 3]
```diff
@@ -820,7 +820,7 @@ def __init__(
 
         self.topk = topk
 
-    def ensure_exactly_one_mask_token(self, masked_index: Union[torch.Tensor, np.ndarray]):
```
Comment (julien-c) [inline src/transformers/pipelines.py]: You can use "quotes" here to prevent imports, that's what we do elsewhere

Comment (mfuntowicz) [reply src/transformers/pipelines.py]: Didn't know this one! Thanks 🙏

**Review by LysandreJik — APPROVED** [review]
LGTM! Why don't we handle multiple tokens, by the way? The models should behave somewhat okay given that their pre-training hides more than one token, right?

**Comment by julien-c** [comment]
no particular reason @LysandreJik other than developer lazyness (me)

**Comment by zanderbush** [comment]
> no particular reason @LysandreJik other than developer lazyness (me)

@julien-c 

Hi Julien. I understand if the implementation of multiple tokens is too inconvenient, but I am sure that many would appreciate such a feature.

**Comment by yashkant** [comment]
Hey @mfuntowicz , @julien-c,

> LGTM! Why don't we handle multiple tokens, by the way? The models should behave somewhat okay given that their pre-training hides more than one token, right?

I am implementing this feature in my local version, is there anything I should be careful of? I am just planning to remove the exception trigger and handle the multiple masked tokens output. 

Also, let me know if you'd like a PR for this.

**Pull request merged**